### ISSUE TITLE
Request to update image to use latest Jenkins LTS - 2.73.2

### ISSUE THREAD

**Issue by scoheb** [openshift/jenkins#397]
This will address some security issues: https://jenkins.io/security/advisory/2017-10-11/

**Comment by bparees** [comment]
@smunilla can we get jenkins 2.73.2 packaged?

@coreydaley https://github.com/openshift/jenkins/blob/master/2/Dockerfile.rhel7#L15 and https://github.com/openshift/jenkins/blob/master/2/Dockerfile.rhel7#L23 should have been updated when you moved the rhel image to 2.73.1.

**Comment by bparees** [comment]
bumping the centos image here: https://github.com/openshift/jenkins/pull/398

also fixing the rhel7 dockerfile.

but we'll need @smunilla to package the 2.73.2 release before the rhel image can be updated.

**Comment by scoheb** [comment]
I think this can be closed...at least for Centos.
Thanks!

**Comment by bparees** [comment]
yeah i want to keep it open until we get rhel updated.  @smunilla can you ack that?

**Comment by gabemontero** [comment]
A new advisory just came out:  https://jenkins.io/blog/2017/11/08/security-updates/

There is now a 2.73.3

**Comment by smunilla** [comment]
@bparees @gabemontero I'll work on getting us a new build this week.

**Comment by smunilla** [comment]
@bparees @gabemontero Correction: I built it now. It'll make it into tonight's build.

**Comment by bparees** [comment]
bumping centos to 2.73.3 here: https://github.com/openshift/jenkins/pull/413

rhel will go to 2.73.3 once @smunilla finishes building it today (and we ship a new jenkins rhel image with 3.7 of course).

**Comment by bparees** [comment]
next rhel image will have 2.73.3.

**Comment by mike4263** [comment]
@bparees - Can you check whether this has been updated?  It appears to still be 2.46 as of the 3.9 release.

https://access.redhat.com/containers/?tab=tech-details#/registry.access.redhat.com/openshift3/jenkins-2-rhel7

**Comment by bparees** [comment]
looks like 2.89 to me, @mike4263 
```
$ docker run -it registry.access.redhat.com/openshift3/jenkins-2-rhel7:v3.9 /bin/sh
Digest: sha256:889b6746b0a2041a119d2e65e682f7614d230a5cab14200e20c497ae9f863c83
Status: Downloaded newer image for registry.access.redhat.com/openshift3/jenkins-2-rhel7:v3.9
sh: cannot set terminal process group (-1): Inappropriate ioctl for device
sh: no job control in this shell
sh-4.2$ rpm -qa | grep jenkins
jenkins-2.89.4.1519670652-1.el7.noarch
jenkins-2-plugins-3.9.1519779801-1.el7.noarch
```

**Comment by mike4263** [comment]
I had been working on :latest as of 10 days ago.  So, this was never updated for 3.7?

Related question - how is the metadata for those images defined?  That is still saying 2.46 on the registry.access.redhat.com site.

Just trying to understand the build process here.

**Comment by bparees** [comment]
@mike4263 the "latest" tag is supposed to always point to v3.6, so if you have been pulling by that, you would have only seen jenkins v2.46 since that is what v3.6 included.  (the fact that the latest tag is now matching the v3.9 tag is actually a mistake that needs to be corrected).

The v3.7 tag includes v2.89:
```
$ docker run -it registry.access.redhat.com/openshift3/jenkins-2-rhel7:v3.7 /bin/sh
sh-4.2$ rpm -qa | grep jenkins
jenkins-2.89.2-1.el7.noarch
jenkins-2-plugins-3.7.1510081324-1.el7.noarch
```

(which is actually somewhat surprising to me, i'd have expected it to be 2.73 as I believe 2.73 was the LTS available when OCP v3.7 shipped.  The centos v3.7 image is on 2.73.  @gabemontero, possibly worth investigating how/why that happened)


As for the metadata, that label comes from here:
https://github.com/openshift/jenkins/blob/openshift-3.9/2/Dockerfile.rhel7#L24

and again it appears something is out of sync between what we're providing, and what got published, @gabemontero (our rhel7 dockerfile sets the label as jenkins 2, but it would appear the dist-git repo has it as jenkins 2.46)


In sum:

1) you should always pull jenkins images by a versioned tag, not latest
2) look at the rpm levels, not the metadata, to confirm what's really in there (but we'll fix the metadata)
3) jenkins images ship w/ OCP and contain the latest LTS available at the time OCP ships
4) the "latest" tag is currently pointing to the wrong thing(v3.9 image), should point to v3.6 image(and always will, once it is fixed.  this is for backwards compatibility reasons)
5) the display name label metadata (jenkins 2.46) is currently wrong but basically meaningless
6) the jenkins v3.7 image appears to contain a newer LTS(2.89) than i would have expected (2.73)

**Comment by gabemontero** [comment]
OK, summarizing some offline conversations for bookkeeping here:

1) looks like @jupierce update 3.6 and 3.8 (which was not shipped) to 2.89.2 earlier this year with his update job  ... assuming that somehow bumped 3.7 as well ... at this time, @bparees and I are in agreement we'll just leave it as is, but we are not planning to bump the centos 3.6 and 3.7 images at this time .... we'll circle back with @jupierce when he returns from PTO on why he did this

2) worked some with @adammhaile and confirmed the label is currently hardcoded to 2.46 acroos all the releases in the distgit side of things.  He is opening PRs to address that and updates to the images for 3.6, 3.7, and 3.9 should go out in the next set of errata for each of those releases.

**Comment by mike4263** [comment]
@bparees  Ah, thanks for clarifying.
